src/components/Comments/Comment/Comment.js

```diff
@@ -78,6 +80,7 @@ Comment.propTypes = {
     content: string.isRequired,
     username: string.isRequired,
     createdAt: string.isRequired,
+    label: string.isRequired,
```

Comment:
`label` is not part of `comment`. You should move line 83 outside of `comment` object.

Reply:
I agree